apps/antalmanac/src/stores/AppStore.ts

```diff
@@ -223,6 +232,37 @@ class AppStore extends EventEmitter {
             return false;
         }
         this.unsavedChanges = false;
+
+        if (window.localStorage.getItem('unsavedAction') != null) {
+            if (confirm('You have unsaved changes. Would you like to load them?')) {
+                // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                // @ts-ignore
+                const unsavedAction = JSON.parse(window.localStorage.getItem('unsavedAction'));
+                for (const action of unsavedAction) {
+                    if (action[0] == 'addCourse') {
+                        this.schedule.addCourse(action[1], action[2]);
+                    } else if (action[0] == 'deleteCourse') {
+                        this.schedule.deleteCourse(action[1], action[2]);
+                    } else if (action[0] == 'addCustomEvent') {
+                        this.schedule.addCustomEvent(action[1], action[2]);
+                    } else if (action[0] == 'deleteCustomEvent') {
+                        this.schedule.deleteCustomEvent(action[1]);
+                    } else if (action[0] == 'editCustomEvent') {
+                        this.schedule.editCustomEvent(action[1], action[2]);
+                    } else if (action[0] == 'changeCustomEventColor') {
+                        this.schedule.changeCustomEventColor(action[1], action[2]);
+                    } else if (action[0] == 'changeCourseColor') {
+                        this.schedule.changeCourseColor(action[1], action[2], action[3]);
+                    } else if (action[0] == 'clearSchedule') {
+                        this.schedule.clearCurrentSchedule();
+                    } else if (action[0] == 'copySchedule') {
+                        this.schedule.copySchedule(action[1]);
+                    }
```

Comment:
I think a `switch` statement is better here. Alternatively, use a giga-mega-uber nested ternary /joke